lib/ilp.js

```diff
@@ -0,0 +1,110 @@
+const { createConnection } = require('ilp-protocol-stream')
+const SPSP = require('ilp-protocol-spsp')
+const Plugin = require('ilp-plugin-btp')
+
+class Ilp {
+  constructor ({ log }) {
+    this.log = log
+    // the address of the internal NLB that the lambda will call to connect to the ILP connector
+    this.ilpConnectorAddr = process.env.ILP_CONNECTOR_ADDR
+  }
```

Comment:
I think we should pass in the addr as config to this class; will make testing easier/clearer

something like:

```suggestion
  constructor ({ config, log }) {
    this.config = config
    this.log = log
    // the address of the internal NLB that the lambda will call to connect to the ILP connector
    this.ilpConnectorAddr = config.getIlpConnectorAddress()
  }
```

and config can pull it from `this.env` which is `process.env` by default